app/ui-react/syndesis/src/app/App.tsx

```diff
@@ -58,6 +58,11 @@ export const App: React.FunctionComponent<IAppBaseProps> = ({
     }
   };
 
+  const spinner = document.getElementById('loading-page-spinner');
```

Comment:
This block would run at every render, you can run it once like this

```
React.useEffect(() => {
  const spinner = document.getElementById('loading-page-spinner')!; //  ! because we know the element will be there
  spinner.setAttribute('hidden', 'true');
}, []);
```

Reply:
Great point, thanks for the suggestion -- updated